<launch>
	<arg name="image_sub_topic_name" default="/kinect/qhd/image_color_rect" />
	<arg name="debug" default="false" />
	<arg name="use_gpu" default="true" />
	<arg name="use_compressed_image" default="false" />

	<node pkg="rail_face_detector" type="face_detector_node.py" name="face_detector_node" output="screen">
		<param name="image_sub_topic_name" type="string" value="$(arg image_sub_topic_name)" />
		<param name="debug" type="bool" value="$(arg debug)" />
    <param name="use_gpu" value="$(arg use_gpu)" />
    <param name="use_compressed_image" value="$(arg use_compressed_image)" />
	</node>
</launch>
